static/docs/commands-reference/import.md

```diff
@@ -21,28 +12,288 @@ Supported schemes:
       out            Output
 ```
 
+## Description
+
+In some cases it is convenient to add a data file to a workspace such that it
+will be automatically updated when the data source is updated. One project might
+produce occasional data files that are used in other projects, for example. Or
+a government agency might produce occasionally updated data of use in a project.
+
+DVC supports `.dvc` files which refer to an external data file, see
+[External Dependencies](/doc/user-guide/external-dependencies). In such a DVC
+file, the `deps` section lists a remote file specification, and the `outs`
+section lists the corresponding local file name in the workspace. It records
+enough data from the remote file to enable DVC to efficiently check the remote
+file to determine if the local file is out of date.  DVC uses this data to then
+download the file to the workspace, and to re-download it upon changes.
+
+The `dvc import` command helps the user create such an external data dependency.
+
+DVC supports several types of remote files:
+
+Type | Discussion | URL format
+-----|------------|------------
+`local` | Local path | `/path/to/local/file`
+`s3` | Amazon S3 | `s3://mybucket/data.csv`
+`gs` | Google Storage | `gs://mybucket/data.csv`
+`ssh` | SSH server | `ssh://user@example.com:/path/to/data.csv`
+`hdfs` | HDFS | `hdfs://user@example.com/path/to/data.csv`
+`http` | HTTP to file with _strong ETag_ | `https://example.com/path/to/data.csv`
```

Comment:
let's make strong ETag a link. It would be great to mention that it's required to tracked changes.